MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -426,22 +415,4 @@ + (void)resetSharedInstance {
   sharedInstance = nil;
 }
 
-#pragma mark - Application life cycle
```

Comment:
I like that our umbrella doesn't do any resume/suspend tasks. It's job is to just get things up and running, not worry about state of the sending pipeline.